### ISSUE TITLE
Explicitly ignore extension modules in twistedchecker script.

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8910]
|[<img alt="tomprince's avatar" src="https://avatars.githubusercontent.com/u/283816?s=50" width="50" height="50">](https://github.com/tomprince)| @tomprince reported|
|-|-|
|Trac ID|trac#8910|
|Type|enhancement|
|Created|2016-11-19 01:28:54Z|

twistedchecker blows up on extension modules. This explicitly ignores the ones in twisted's tree. See https://github.com/twisted/twistedchecker/issues/118 for the upstream bug.

<details><summary>Searchable metadata</summary>

```
trac-id__8910 8910
type__enhancement enhancement
reporter__tomprince tomprince
priority__normal normal
milestone__None None
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1479518934827089 1479518934827089
changetime__1479576492144541 1479576492144541
version__None None
owner__tomprince tomprince

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="tomprince's avatar" src="https://avatars.githubusercontent.com/u/283816?s=50" width="50" height="50">](https://github.com/tomprince)<a name="note_1"></a>|@tomprince commented|
|-|-|

https://github.com/twisted/twisted/pull/598

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_3"></a>|@adiroiban set owner to @tomprince|
|-|-|

Thanks for the update!

I think that we should merge it.

Once twistedcheker/pylint is updated to ignore non python files we can remove it from Twisted.

Please merge. Thanks!

**Comment by twisted-trac** [comment]
|<img alt="Tom Prince's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_4"></a>|Tom Prince set status to `closed`|
|-|-|

In changeset 5a8299afb57a6988ab6010be1f98c945ce46c3de
```
#!CommitTicketReference repository="" revision="5a8299afb57a6988ab6010be1f98c945ce46c3de"
Merge tomprince/twistedchecker-exceptions: Explictly ignore exension modules in twistedchecker.

Author: tomprince
Reviewer: adiroiban
Fixes: #8910

twistedchecker blows up on extension modules. This explicitly ignores the ones in twisted's tree. Once twistedchecker and pylint are fixed, this can be removed.
Explicitly ignore extension modules.
```
